cpp/src/barretenberg/dsl/acir_format/ecdsa_secp256k1.cpp

```diff
@@ -144,6 +143,7 @@ void dummy_ecdsa_constraint(Builder& builder, EcdsaSecp256k1Constraint const& in
     std::vector<uint32_t> pub_x_indices_;
     std::vector<uint32_t> pub_y_indices_;
     std::vector<uint32_t> signature_;
+    std::vector<uint32_t> message_indicies_;
```

Comment:
Not sure why this new parameter is needed

Reply:
Tested locally and not including the hashed_message_indices, passes for secp256k1 but fails for secp256r1. This is either a problem in our tests or a problem with barretenberg. Since we copied, the tests line for line, I'm inclined to believe its the latter. 

Lets open up an issue for this referecning this comment and merge as is, since its orthogonal to this PR

Reply:
Issue opened here: https://github.com/AztecProtocol/barretenberg/issues/591